cpp-fs/storage/test/test_storage.cpp

```diff
@@ -0,0 +1,66 @@
+#include <gtest/gtest.h>
+
+#include "error_types.h"
+#include "partition/in_memory_partition.hpp"
+#include "storage.hpp"
+
+namespace tests::storage {
+
+namespace {
+template <typename T>
+  requires std::is_default_constructible_v<T>
+
```

Comment:
This empty line looks very strange

Reply:
We will need to switch to a newer version of the clang format, the one we are currently using forces us to insert line breaks